esyi/Sandbox.ml

```diff
@@ -121,40 +133,46 @@ let ofDir ~cfg (path : Path.t) =
       Manifest.toPackage ~version manifest
     in
 
+   let%bind origin = originOfPath path in
```

Comment:
I did it this way mainly because we have a path to the project directory and we don't know whether the manifest file is "esy.json" or "package.json", and `Manifest.find` determines which it is, and so we need to unpack its result one way or another. Do we only want to support this for package.json just add it to the path like `let origin = Esy(Path.(path / "package.json"))`?. Let me know what you prefer.